test/deploy/rackhd_source_install.py

```diff
@@ -108,7 +94,7 @@ def test02_clone_rackhd_source(self):
                                                  + fit_common.fitinstall()['rackhd']['branch']
                                                  )['exitcode'], 0, "Branch not found on RackHD repo.")
         # clone modules
-        for repo in modules:
+        for repo in fit_common.fitinstall().keys():
```

Comment:
@johren  These were intended for future. I had put in some work to create a new playbook to support this deployment but that got derailed. How about I comment this out with a note.